targets/targets.json

```diff
@@ -2622,6 +2622,9 @@
             "FLASH",
             "MPU"
         ],
+        "device_has_remove": [
+            "SPI_ASYNCH"
```

Comment:
We don't see any reason to remove SPI_ASYNC for this specific target...

Reply:
The main reason is that async mode does not work on this target (test case fails). 
The best option would be to find why. I will try to do further investigation, but I wanted first to check if maybe you know something about this.

Reply:
@ABOSTM